addons/hr_timesheet/models/hr_employee.py

```diff
@@ -24,3 +26,14 @@ def action_unlink_wizard(self):
             'target': 'new',
             'context': self.env.context,
         }
+
+    def action_timesheet_from_employee(self):
+        action = self.env["ir.actions.act_window"]._for_xml_id("hr_timesheet.timesheet_action_from_employee")
+        context = action['context'].replace('active_id', str(self.id))
+        context = literal_eval(context)
+        context.update({
+            'create': self.active,
+            'edit': self.active,
```

Comment:
Isn't it possible that one of those keys is already in the context?
```suggestion
            'create': context.get('create') and self.active,
            'edit': context.get('edit') and self.active,
```
:man_shrugging:

Reply:
Doing this mean user will not able to create/edit timesheets for active employees because even though create/edit is default true its not always there in context.
We could use ` context.get('create') or self.active` if we want user to create timesheets if create(True) is there in context but I don't think we want that.
so should I skip this comment?